app/lib/backend/storage.rb

```diff
@@ -7,8 +7,7 @@ def self.parse_config
       config = File.read("#{Rails.root}/config/backend_redis.yml")
       config = ERB.new(config).result(binding)
       config = YAML.load(config)
-      config = config.fetch(Rails.env).symbolize_keys
-      config.reverse_merge(logger: Rails.logger)
+      config.fetch(Rails.env).deep_symbolize_keys
```

Comment:
why removed the `reverse_merge`?

Reply:
In fact, tests fail.

Reply:
Confirmed, the `:logger` param breaks the redis client when porta tries to access to backend DB directly, for instance, to get the amount of `hits` for an application. If you add `config.reverse_merge(logger: Rails.logger)` back and try to open an App screen, you'll see the error in the console when it tries to retrieve the usage data.

Reply:
So can we in some other way set logger? Or is this not needed anymore?

Reply:
`redis-rb` doesn't provide a way to set a logger after version 5.0. Check this: https://github.com/redis/redis-rb/issues/1182

Reply:
So basically there is no logging now. Cool. No feature, no problems.